components/fxa-client/examples/oauth_flow.rs

```diff
@@ -6,7 +6,7 @@ use url::Url;
 const CONTENT_SERVER: &str = "http://127.0.0.1:3030";
 const CLIENT_ID: &str = "7f368c6886429f19";
 const REDIRECT_URI: &str = "https://mozilla.github.io/notes/fxa/android-redirect.html";
-const SCOPES: &[&str] = &["https://identity.mozilla.com/apps/oldsync"];
+const SCOPES: &[&str] = &["profile"];
```

Comment:
Out of curiosity, what motivated this change?

Reply:
Using "https://identity.mozilla.com/apps/oldsync" didn't work with the redirection, whereas "profile" did.